src/installation.rst

```diff
@@ -237,12 +237,13 @@ To do so create a symbolic link to the wrapper, for each of these commands:
 Configuration
 -------------
 
-Cylc uses "sane and safe" defaults and is suitable for use "out of the box".
+Cylc uses "sane and safe" defaults and is suitable for use "out of the box",
```

Comment:
Mr self deprecating suggests that we have a bit of faith in our product. The inverted commas imply that we don't fully believe that these settings are sane and safe.

```suggestion
Cylc uses sane and safe defaults and is suitable for use "out of the box",
```